### PR TITLE
Register MicronTracker stereo camera as a video source

### PR DESCRIPTION
**Pull request by auneri**
Very basic support that allows streaming to a client (e.g., 3D Slicer) through OpenIGTLink.

### CODE REVIEW

**Comment by auneri** [comment]
Oh and I also had to remove copying of nonexistent [digiclops](https://github.com/PlusToolkit/PlusLib/blob/Plus-2.6/src/PlusDataCollection/CMakeLists.txt#L402) and [triclops](https://github.com/PlusToolkit/PlusLib/blob/Plus-2.6/src/PlusDataCollection/CMakeLists.txt#L408) libraries for the version of MTC (v3.7.6.8) I have.

**Comment by adamrankin** [comment]
@lassoan what's the precedent for tracker devices that also produce video?

**Comment by lassoan** [comment]
It should be no problem to have a device that can produce both tracking and imaging data. It would probably make sense to have the tracking data, left eye video (and right eye video, if needed) in separate channels, so that tracking data can be interpolated and fused with a different image stream.

File: src/PlusDataCollection/MicronTracking/vtkPlusMicronTracker.cxx [hunk 1]
```diff
@@ -310,6 +335,8 @@ PlusStatus vtkPlusMicronTracker::GetImage(vtkImageData* leftImage, vtkImageData*
 
   int imageWidth = this->MT->mtGetXResolution(-1);
```
Comment (adamrankin) [inline src/PlusDataCollection/MicronTracking/vtkPlusMicronTracker.cxx]: Can this function ever return < 0?

File: src/PlusDataCollection/MicronTracking/vtkPlusMicronTracker.cxx [hunk 2]
```diff
@@ -240,6 +246,25 @@ PlusStatus vtkPlusMicronTracker::InternalUpdate()
 #endif
   }
 
+  this->GetImage(this->FrameLeft, this->FrameRight);
+  vtkPlusDataSource* aSource(NULL);
+  for (int i = 0; i < this->GetNumberOfVideoSources(); ++i)
+  {
+    if (this->GetVideoSourceByIndex(i, aSource) != PLUS_SUCCESS)
```
Comment (adamrankin) [inline src/PlusDataCollection/MicronTracking/vtkPlusMicronTracker.cxx]: Please add a NotifyConfigure overload that checks that the user has created 2 and exactly 2 video sources for image output.

**Comment by adamrankin** [comment]
Closing as #442 accomplishes the same goal

**Pull request closed without merging**